### ISSUE TITLE
Support edge lengths in `UnrootTree()`

### ISSUE THREAD

**Issue by marwa38** [ms609/TreeTools#89]
I was wondering what exactly happen the tree while unrooting it in the background?
I have got some phylogenetic alpha diversity to run on on phyloseq objects with rooted trees which I need to unroot first before running this code as it was advised by some posts to avoid the random rooting of the tree 
Does it make changes in any way to the tree?

```
### ROOTING the tree more appropriately ####
pick_new_outgroup <- function(tree.unrooted){
  require("magrittr")
  require("data.table")
  require("ape") # ape::Ntip
  # tablify parts of tree that we need.
  treeDT <- 
    cbind(
      data.table(tree.unrooted$edge),
      data.table(length = tree.unrooted$edge.length)
    )[1:Ntip(tree.unrooted)] %>% 
    cbind(data.table(id = tree.unrooted$tip.label))
  # Take the longest terminal branch as outgroup
  new.outgroup <- treeDT[which.max(length)]$id
  return(new.outgroup)
}

new.outgroup <- pick_new_outgroup(tree.unrooted)
# > new.outgroup
# [1] "ASV679"

rootedTree <- ape::root(phy_tree(ps.3), outgroup = new.outgroup, resolve.root = TRUE)
```
Why I am asking? because I have been woking with the above codes without any errors or warnings
but when I unrooted the tree using unrootTree() I get this error 
```
> new.outgroup <- pick_new_outgroup(unrooted.tree)
Warning message:
In as.data.table.list(x, keep.rownames = keep.rownames, check.names = check.names,  :
  Item 1 has 3839 rows but longest item has 3840; recycled with remainder.
```
Thanks

**Comment by ms609** [comment]
Edge lengths are not yet supported (see `?UnrootTree`).  This is something I may add in a future version of the package if there is demand.

**Comment by ms609** [comment]
Merged into #49.
